gulpfile.js

```diff
@@ -66,7 +66,7 @@ gulp.task('deploy', function () {
             css: 'css',
             sass: 'scss',
             sourcemap: false,
-            style: 'compressed'
+            style: 'expanded'
```

Comment:
why?

Reply:
Because someone in core unminified the file, modified and commited like that, so now when we change anything here and compiled minified there is no way for us to see if we're overwriting changes made in core, so for that I rather to do this so we can always make sure that the changes we're pushing for core are right.